packages/workflow/src/WorkflowErrors.ts

```diff
@@ -28,6 +29,7 @@ export class SubworkflowOperationError extends WorkflowOperationError {
 
 	constructor(message: string, description: string) {
 		super(message);
+		this.severity = 'warning';
```

Comment:
since `SubworkflowOperationError` extends `WorkflowOperationError`, this line is redundant.